bin/sdk-cli.js

```diff
@@ -62,10 +62,13 @@ const build = ( target, argv ) => {
 
 		if ( stats.hasErrors() ) {
 			console.log( chalk.red( 'Built with errors' ) );
+			process.exitCode = 1;
```

Comment:
Would it make sense to `process.exit( 1 )` instead (like we're doing on [L58](https://github.com/Automattic/wp-calypso/pull/30019/files#diff-8a959d6bed3d677b399911293defd68fR58) already)? No code is run after this, is there?

```suggestion
			process.exit( 1 );
```

Reply:
(Same for the 0 exit codes below, obv)

Reply:
Related; should we _not_ `process.exit(1)` if the CLI is running in watch-mode?

Reply:
Ignore that, response was already elsewhere in the thread: https://github.com/Automattic/wp-calypso/pull/30019#issuecomment-476635176